osidb/models.py

```diff
@@ -640,22 +640,6 @@ class FlawRequiresSummary(models.TextChoices):
     # mitigation to apply if the final fix is not available, from srtnotes "mitigation"
     mitigation = models.TextField(blank=True)
 
-    def is_major_incident_temp(self):
```

Comment:
Would it make sense to keep this, make it a property, rename it to `is_major_incident` (replacing the old field) so that we can use it in other parts of code instead of having to type
```py
if flaw.major_incident_state in [
    Flaw.FlawMajorIncident,
    Flaw.FlawMajorIncident.CISA_APPROVED,
]
```
each time?

Reply:
I discussed it with @osoukup. The property was too ambiguous (it also contained the incorrect `REQUESTED` state) so we decided to get rid of it.

Reply:
Yeah. When someone sees `is_major_incident` the usual understanding is to assume it means `APPROVED` which was not true and therefore quite confusing. The property was brought from the old tooling where it was some kind of shortcut but going forward we should start distinguishing which type of MI it is and whether only requested or already approved.